sunkit_image/tests/test_offlimb_enhance.py

```diff
@@ -1,12 +1,106 @@
-import os
-
 import numpy as np
 import pytest
 import astropy.units as u
-from numpy.testing import assert_allclose
 
 import sunpy
 import sunpy.map
 import sunpy.data.test
+import sunkit_image.utils.utils as utils
+import sunkit_image.offlimb_enhance as off
+
+
+@pytest.fixture
+def map_test1():
+        x = np.linspace(-2, 2, 5)
+        grid = np.meshgrid(x, x.T)
+        test_data1 = np.sqrt(grid[0]**2 + grid[1]**2)
+        test_data1 *= 10
+        test_data1 = 28-test_data1
+        test_data1 = np.round(test_data1)
+        header = {}
+        test_map1 = sunpy.map.Map((test_data1, header))
+        return test_map1
+
+
+@pytest.fixture
+def map_test2():
+        x = np.linspace(-2, 2, 5)
+        grid = np.meshgrid(x, x.T)
+        test_data1 = np.sqrt(grid[0]**2 + grid[1]**2)
+        test_data1 *= 10
+        test_data1 = 28-test_data1
+        test_data1 = np.round(test_data1)
+        header = {}
+        test_data2 = np.where(test_data1[:, 0:2] == 6, 8, test_data1[:, 0:2])
+        test_data2 = np.concatenate((test_data2, test_data1[:, 2:]), axis=1)
+        test_map2 = sunpy.map.Map((test_data2, header))
+        return test_map2
+
```

Comment:
What is the difference between the two test maps?

Reply:
Both the maps have a radial profile but there is a subtle difference.
```test_map1``` has all the elements in a circular ring equal to each other.
```
array([[-0.,  6.,  8.,  6., -0.],
       [ 6., 14., 18., 14.,  6.],
       [ 8., 18., 28., 18.,  8.],
       [ 6., 14., 18., 14.,  6.],
       [-0.,  6.,  8.,  6., -0.]])
```
```test_map2``` has one circular ring where all elements are not equal.
```
array([[-0.,  8.,  8.,  6., -0.],
       [ 8., 14., 18., 14.,  6.],
       [ 8., 18., 28., 18.,  8.],
       [ 8., 14., 18., 14.,  6.],
       [-0.,  8.,  8.,  6., -0.]])
```

Reply:
What are the results from the algorithm when it is applied to both of these maps?

Reply:
The results are already written in the test and they turn out to be the same when I manually calculated the results. But please have a closer look just to be sure I did not commit a mistake.